doc/principe.md

```diff
@@ -0,0 +1,38 @@
+
+# Principe de fonctionnement du validateur de document
+
+## 1) Chargement du modèle de document
+
+Le modèle de document est chargé en fonction des options `--config/--version`.
```

Comment:
I think it lacks details. We should be more specific about what each of both options does.

Reply:
`--model=<DOCUMENT_MODEL_URL>` describe to simplify (`--config/--version` is the legacy way)